src/ray/common/constants.h

```diff
@@ -18,7 +18,7 @@
 #include <stdint.h>
 
 /// Length of Ray full-length IDs in bytes.
-constexpr size_t kUniqueIDSize = 28;
+constexpr size_t kUniqueIDSize = 32;
```

Comment:
Is it necessary to update this? 28B seems more than enough. Can there be unintended breakages?